HW02_B_Mimic.md

```diff
@@ -1,89 +1,118 @@
-HW02\_B\_Graph-Mimic

```

Comment:
So technically you should have edited the rmd file called "starter", and then knitted that file which would have created the correct .md file. For some reason doing it this way makes it so we can't see the graphs on git